src/main/java/com/backend/blooming/goal/domain/Teams.java

```diff
@@ -78,4 +80,20 @@ public boolean isTeamAndAccepted(final User user) {
         return goalTeams.stream()
                         .anyMatch(goalTeam -> goalTeam.getUser().equals(user) && goalTeam.isAccepted());
     }
+
+    public void deleteUser(final Long userId) {
+        this.goalTeams.forEach(goalTeam -> {
+            if (goalTeam.getUser().getId().equals(userId)) {
+                goalTeam.updateDeleted();
+            }
+        });
```

Comment:
개인적으로 아래와 같은 for문을 통해 break를 추가하면 어떨까 하는 생각이 있습니다.
현재는 물론 최대 5명이기에 상관없긴 하지만, 팀원이 많아질 수 있다면 결국 성능상 아주 조금은 낮아지지 않을까 해서요!
물론 정말 완벽한 성능 개선을 생각한다면 좀 더 고민해 봐야 하긴 하지만요...ㅎㅎ

```suggestion
        for (GoalTeam goalTeam : goalTeams) {
            if (goalTeam.getUser().getId().equals(userId)) {
                goalTeam.updateDeleted();
                break;
            }
        }
```

Reply:
위에서 코멘트 남긴대로 for문 돌다가 if문에서 걸리면 goalTeam delete 해주고 Teams에서는 remove해주는 식으로 구현했습니다. 감사합니다!